kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterService.java

```diff
@@ -48,6 +50,13 @@ static String serviceName(Clusters cluster) {
         return cluster.getName();
     }
 
+    /**
+     * @return the fully qualified service hostname
+     */
+    static String absoluteServiceHost(@NonNull KafkaProxy primary, @NonNull Clusters cluster) {
```

Comment:
The annotations on the package (`package-info.java`)  should mean `@NonNull` is the default.